app/test/fixture-helper.ts

```diff
@@ -50,6 +50,8 @@ export function setupFixtureRepository(repositoryName: string): string {
 export async function setupEmptyRepository(): Promise<Repository> {
   const repoPath = temp.mkdirSync('desktop-empty-repo-')
   await GitProcess.exec([ 'init' ], repoPath)
+  await GitProcess.exec([ 'config' , 'user.name', 'Lucas Ding' ], repoPath)
```

Comment:
[This is also a bug that Git fixed with 2.13.0](https://github.com/git/git/blob/master/Documentation/RelNotes/2.13.0.txt#L383-L384)